### DISCUSSION TITLE
How to remove a commit from a PR in the upstream repo, after removal of the commit from my fork?

### DISCUSSION THREAD

**Discussion by grahamperrin** [community/community#60107] [Pull Requests] [answered]
I made changes to the `2023q2` branch in _my fork of_ a FreeBSD repo, then remembered an open pull request – in the FreeBSD repo (not in my fork) – for unrelated changes. 

I performed a hard reset (to remove the commit from the branch in my fork), then force-pushed: 

```text
% git -C /usr/doc switch 2023q2
branch '2023q2' set up to track 'origin/2023q2'.
Switched to a new branch '2023q2'
% git -C /usr/doc log -n 5 --oneline
f10fb7c7f5 (HEAD -> 2023q2, origin/2023q2) boot performance: BSDCan session title
ae15792c0d Merge branch 'freebsd:main' into 2023q2
3db500c1fa (origin/main, origin/HEAD, freebsd/main, main) status: Add 2023Q2 report from clusteradm
f90edb875e Status/2023Q2/openstack.adoc: Add report
c6d6c48402 Status/2023Q2/mfsdbsd.adoc: Add report
% git -C /usr/doc reset --hard HEAD^1
HEAD is now at ae15792c0d Merge branch 'freebsd:main' into 2023q2
% git -C /usr/doc status
On branch 2023q2
Your branch is behind 'origin/2023q2' by 1 commit, and can be fast-forwarded.
  (use "git pull" to update your local branch)

nothing to commit, working tree clean
% git -C /usr/doc push --force
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
To https://github.com/grahamperrin/freebsd-doc.git
 + f10fb7c7f5...ae15792c0d 2023q2 -> 2023q2 (forced update)
% 
```

– then realised the effect ~~of my mistake~~: 

- whilst the commit is [no longer in any branch (in my fork)](https://github.com/grahamperrin/freebsd-doc/commit/f10fb7c7f5823c0bfe878ddfc1e7710c7726e3f0)
- the commit [does remain in the PR](https://github.com/freebsd/freebsd-doc/pull/191/commits). 

### Question

How can I remove the commit from the PR?

**Comment by GhostOf0days** [comment, marked as answer]
Hello. This is likely an issue with the ongoing [incident with pull requests](https://www.githubstatus.com/incidents/pr0ptqcw6q8d). Reverting a commit and force pushing that revert on your a branch in your fork that was used to make the PR normally does force the PR to go back to the commit you reverted to.

**Reply by GhostOf0days** [reply]
I would not try messing with the PR until the incident with pull requests is fixed.

**Reply by grahamperrin** [reply]
Thank you, 

> … ongoing [incident with pull requests](https://www.githubstatus.com/incidents/pr0ptqcw6q8d). …

The commit remained in the PR for more than four hours after the reported resolution of the incident. 

I reviewed thirteen hours after the reported resolution, the PR was better.
